### PR TITLE
Update missing Chinese translations of v2.8 and latest versions

### PR DESCRIPTION
**Pull request by dingluyy**
Description
---
Update Chinese translations of the latest and v2.8 versions.

Modified files of version latest & v2.8:

- integrations-in-rancher/cloud-marketplace/aws-cloud-marketplace/aws-cloud-marketplace.md
- integrations-in-rancher/cloud-marketplace/cloud-marketplace.md
- integrations-in-rancher/elemental/elemental.md
- integrations-in-rancher/fleet/architecture.md
- integrations-in-rancher/fleet/fleet.md
- integrations-in-rancher/fleet/overview.md
- integrations-in-rancher/fleet/use-fleet-behind-a-proxy.md
- integrations-in-rancher/fleet/windows-support.md
- integrations-in-rancher/harvester/harvester.md
- integrations-in-rancher/harvester/overview.md
- integrations-in-rancher/integrations-in-rancher.mdx
- integrations-in-rancher/kubernetes-distributions/kubernetes-distributions.md
- integrations-in-rancher/kubewarden/kubewarden.md
- integrations-in-rancher/longhorn/longhorn.md
- integrations-in-rancher/longhorn/overview.md
- integrations-in-rancher/neuvector/neuvector.md
- integrations-in-rancher/neuvector/overview.md
- integrations-in-rancher/rancher-desktop.md

### CODE REVIEW

**Comment by btat** [comment]
Hi @JacieChao, could you or someone from your team please help review this PR? Thank you!

File: i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/harvester/harvester.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/zh/integrations-in-rancher/harvester"/>
+</head>
+
+## Harvester
+
+Harvester 是 Rancher v2.6.1 新增的功能，是基于 Kubernetes 构建的开源超融合基础架构（HCI）软件。Harvester 安装在裸金属服务器上，提供集成的虚拟化和分布式存储功能。虽然 Harvester 使用 Kubernetes 运行，但它不需要用户了解 Kubernetes 概念，这使得它更加用户友好。
+
+## Harvester 与 Rancher
+
+ 凭借 Rancher Prime 和 Harvester，IT 运营商现在可以访问一个企业级的、易于使用的基础设施平台，该平台可以协同管理他们的虚拟机和 Kubernetes 集群。有关支持产品的更多信息，请参阅 [支持矩阵](https://www.suse.com/suse-harvester/support-matrix/all-supported-versions/harvester-v1-2-0/)。通过 Rancher 虚拟化管理功能，用户可以导入和管理多个 Harvester 集群。利用 Rancher 的认证功能和 RBAC 控制来支持多租户。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/harvester/harvester.md:15]: IT 运营商 => IT 运维人员
有关支持产品的更多信息 => 有关产品支持的更多信息

File: i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/harvester/overview.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/zh/integrations-in-rancher/harvester/overview"/>
+</head>
+
+[Harvester](https://docs.harvesterhci.io/) 是 Rancher v2.6.1 新增的功能，是基于 Kubernetes 构建的开源超融合基础架构（HCI）软件。Harvester 安装在裸金属服务器上，提供集成的虚拟化和分布式存储功能。虽然 Harvester 使用 Kubernetes 运行，但它不需要用户了解 Kubernetes 概念，这使得它更加用户友好。
+
+### 功能开关
+
+Harvester 功能开关用于管理对 Rancher 中虚拟化管理（VM）页面的访问，用户可以在该其中直接导航到 Harvester 集群并访问 Harvester UI。Harvester 的功能开关默认启用。如需了解 Rancher 中功能开关的更多详细信息，请单击 [此处](../../how-to-guides/advanced-user-guides/enable-experimental-features/enable-experimental-features.md)。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/harvester/overview.md:13]: 用户可以在该其中直接导航到 => 用户可以直接导航到
[此处]前面不需要增加空格

File: i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/harvester/overview.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+* 如果禁用了 Harvester 功能开关，并且导入了 Harvester 集群，Harvester 集群将显示在集群管理页面的 Rancher 集群列表中。仅当功能开关为关闭时，Harvester 集群才会显示在集群列表中。
+
+* 集成 Harvester 后，你可以将 Harvester 集群导入 Rancher，对应的集群类型是 `Harvester`。
+
+* 用户只能在虚拟化管理页面上导入 Harvester 集群。不支持在集群管理页面上导入集群，并且会出现警告，建议您返回虚拟化管理页面执行此操作。
+
+### Harvester 主机驱动
+
+[Harvester 主机驱动](https://docs.harvesterhci.io/v1.1/rancher/node/node-driver/) 通常可用于 Rancher 中的 RKE 和 RKE2 选项。无论 Harvester 功能开关是否启用，主机驱动都是可用的。请注意，主机驱动默认处于关闭状态。用户只能通过集群管理页面在 Harvester 上创建 RKE 或 RKE2 集群。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/harvester/overview.md:27]: 通常前不需要空格

File: i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/harvester/overview.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+Harvester 允许通过 Harvester UI 上传和显示 `.ISO` 镜像，但 Rancher UI 是不支持的。这是因为 `.ISO` 镜像通常需要额外的设置，这会干扰干净的部署（即无需用户干预），并且它们通常不用于云环境。
+
+如需了解 Rancher 中主机驱动的更多详细信息，请单击 [此处](../../how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/about-provisioning-drivers/about-provisioning-drivers.md#node-drivers)。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/harvester/overview.md:31]: about-provisioning-drivers.md#node-drivers => about-provisioning-drivers#主机驱动

File: i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/harvester/overview.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+### 端口要求
+
+Harvester 集群的端口要求可以在 [此处](https://docs.harvesterhci.io/v1.1/install/requirements#networking) 找到。
+
+此外，其他网络注意事项如下：
+
+- 请务必为 VM VLAN 网络启用物理交换机的 VLAN 中继端口。
+- 按照 [此处](https://docs.harvesterhci.io/v1.1/networking/index) 的网络设置指南进行操作。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/harvester/overview.md:40]: [此处]前后不需要空格

File: i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/harvester/overview.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+对于其他集群（例如 K3s 和 RKE1）的其他端口要求，请参阅 [这些文档](https://docs.harvesterhci.io/v1.1/install/requirements/#guest-clusters)。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/harvester/overview.md:42]: [这些文档]前不需要空格

File: i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/integrations-in-rancher.mdx [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+---
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/zh/integrations-in-rancher"/>
+</head>
+
+import {Card, CardSection} from '@site/src/components/CardComponents';
+import {RocketRegular} from '@fluentui/react-icons';
+
+Prime 是 Rancher 生态系统的企业产品，具有更高的安全性、更长的生命周期和对 Prime 专有文档的访问权限。Rancher Prime 安装资产托管在受信任的 SUSE 注册表上，由 Rancher 拥有和管理。受信任的 Prime 注册表仅包括经过社区测试的稳定版本。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/integrations-in-rancher.mdx:12]: 企业产品 => 企业级产品

File: i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/integrations-in-rancher.mdx [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+
+Prime 还提供生产支持选项，以及根据您的商业需求定制的订阅附加组件。
+
+要了解更多信息并开始使用 Rancher Prime，请访问 [本页](https://www.rancher.com/quick-start)。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/integrations-in-rancher.mdx:16]: 本页前不需要空格

File: i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/kubernetes-distributions/kubernetes-distributions.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+</head>
+
+## K3s
+
+K3s 是一款轻量级、完全兼容的 Kubernetes 发行版，专为一系列用例设计，包括边缘计算、物联网、CI/CD、开发和将 Kubernetes 嵌入到应用程序中。它将系统打包为单个二进制文件，使用 sqlite3 作为默认存储，并提供用户友好的启动器，从而简化了 Kubernetes 的管理。K3s 包括 Local Storage 和负载均衡、Helm Chart 控制器和 Traefik CNI 等基本功能。它最大限度地减少了外部依赖，并提供了精简的 Kubernetes 体验。K3s 于 2020 年 6 月作为沙箱项目捐赠给 CNCF。
+
+### K3s 与 Rancher
+
+- Rancher 允许在一系列平台上轻松配置 K3s，包括 Amazon EC2、DigitalOcean、Azure、vSphere 或现有服务器。
+- Kubernetes 集群的标准 Rancher 管理，包括所有概述 [集群管理功能](../how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup#cluster-management-capabilities-by-cluster-type)。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/kubernetes-distributions/kubernetes-distributions.md:16]: 集群管理功能 前不需要空格，注意格式，下同

File: i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/kubewarden/kubewarden.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
+---
+title: 使用 Kubewarden 进行高级策略管理
+---
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/zh/integrations-in-rancher/kubewarden"/>
+</head>
+
+Kubewarden 是一个策略引擎，用于保护和帮助管理集群资源。它允许通过策略验证和更改资源请求，包括上下文感知策略和验证图像签名。它可以在监视或强制模式下运行策略，并提供集群状态的概述。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/kubewarden/kubewarden.md:9]: 图像 => 镜像

File: i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/kubewarden/kubewarden.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+
+## Kubewarden 和 Rancher 
+ 
+Kubewarden 的上游 Helm Chart 完全集成为 Rancher 应用程序，为安装选项提供了 UI。这些 Chart 还附带了尊重 Rancher 堆栈的默认设置（例如：不监管 Rancher 系统命名空间）以及默认的 PolicyServer 和策略。用户可以访问所有 Kubewarden 功能，并可以通过与 Kubernetes API 交互（例如：使用 kubectl）手动部署 PolicyServer 和策略。
+
+Kubewarden 提供了对已删除的 Kubernetes Pod 安全策略的完全替换。 Kubewarden 还通过增强其安全功能，与最新版本的 Kubernetes 引入的新 Pod Security Admission 功能进行了集成。
+
+## Kubewarden 和 Rancher Prime 
+
+Kubewarden 的可用 Rancher UI 扩展将其集成到 Rancher UI 中。UI 扩展可自动安装和配置 Kubewarden 堆栈，并配置对 SUSE 维护的策略的访问。UI 扩展提供了对现成策略的策划目录的访问。使用 UI 扩展，人们可以浏览、安装和配置这些策略。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/kubewarden/kubewarden.md:27]: Kubewarden 的可用 Rancher UI 扩展将其集成到 Rancher UI 中 => Kubewarden 的 Rancher UI 扩展将其集成到 Rancher UI 中。

File: i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/kubewarden/kubewarden.md [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+
+UI 扩展提供了 Kubewarden 堆栈组件及其行为的概述。这包括对 Kubewarden 指标和跟踪事件的访问。操作员可以了解策略对集群的影响并解决问题。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/kubewarden/kubewarden.md:29]: 解决问题 => 排查问题

File: i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/longhorn/longhorn.md [hunk 13]
```diff
@@ -0,0 +1,15 @@
...
+title: 使用 Longhorn 进行云原生存储
+---
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/zh/integrations-in-rancher/longhorn"/>
+</head>
+
+## Longhorn
+
+Longhorn 是官方的 [云原生计算基金会（CNCF）](https://cncf.io/) 项目，它为 Kubernetes 提供了一个可以在任何地方运行的强大的云原生分布式存储平台。当与 Rancher 相结合使用时，Longhorn 使您可以在 Kubernetes 环境中轻松、快速且可靠地部署高可用的持久块存储。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/longhorn/longhorn.md:11]: 官方的后面不需要空格，注意格式

File: i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/neuvector/neuvector.md [hunk 14]
```diff
@@ -0,0 +1,27 @@
...
+- 部署、管理和保护多个集群。
+- 管理和报告 Rancher 工作负载和节点的漏洞和合规性结果。
+
+## NeuVector Prime 和 Rancher Prime 
+
+Rancher Manager 的 NeuVector UI 扩展可用于并支持 Rancher Prime 和 NeuVector Prime 客户。此扩展提供：
+
+- NeuVector 自动化部署，包括 Rancher Prime NeuVector 扩展仪表板。
+- 访问每个集群的重要安全信息，如关键安全事件、漏洞扫描结果和入口/出口暴露。
+- 集成漏洞（CVE）和合规扫描直接导致 Rancher 资源，如节点，容器和 Pod。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/neuvector/neuvector.md:26]: 直接对 Rancher 资源（例如节点和容器/Pod）进行集成漏洞 (CVE) 和合规扫描

File: i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/rancher-desktop.md [hunk 15]
```diff
@@ -0,0 +1,34 @@
...
+Rancher Desktop 将用于桌面开发和测试云原生应用程序的基本工具捆绑在一起。
+
+如果您在本地计算机上运行适用于云环境的应用程序，则通常需要进行大量准备工作。您需要选择一个容器运行时,安装 Kubernetes 和流行的实用程序，并且可能还需要设置虚拟机。单独安装组件并使它们协同工作可能是一个耗时的过程。
+
+为了降低复杂性，Rancher Desktop 为团队提供了以下主要功能：
+
+- 在 macOS、Linux 和 Windows 操作系统上进行简单易用的安装。
+- K3s，一个易于使用的轻量级 Kubernetes 发行版。
+- 能够在 Kubernetes 版本之间轻松切换。
+- 由 Rancher 支持的基于 GUI 的集群仪表板，用于探索本地集群。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/rancher-desktop.md:19]: 探索 => 浏览

File: i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/cloud-marketplace/aws-cloud-marketplace/aws-cloud-marketplace.md [hunk 16]
```diff
@@ -0,0 +1,36 @@
...
+title: AWS Marketplace 集成
+---
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/zh/integrations-in-rancher/cloud-marketplace/aws-cloud-marketplace"/>
+</head>
+
+## 概述
+
+Rancher 提供与 AWS Marketplace 的集成，允许用户购买 SUSE 的支持合同。当您开始支持更多集群时，这种集成使您可以轻松调整支持需求。
```
[the remark this answers is not in the document]
Comment (dingluyy) [reply i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/cloud-marketplace/aws-cloud-marketplace/aws-cloud-marketplace.md]: 已修改

**Review by JacieChao — APPROVED** [review]
LGTM
Thanks

**Comment by JacieChao** [comment]
Hi, @btat 

Thanks to @dingluyy @ly5156 @jianghang8421 @rootwuj @smallteeths @GGGitBoy @STARRY-S for their translation work. 
All missing Chinese translation files have been supplemented(for Latest and v2.8). The relevant PRs are ready for merge.

cc @Jason-ZW

The relevant PRs are:
- https://github.com/rancher/rancher-docs/pull/1206
- https://github.com/rancher/rancher-docs/pull/1207
- https://github.com/rancher/rancher-docs/pull/1212
- https://github.com/rancher/rancher-docs/pull/1215
- https://github.com/rancher/rancher-docs/pull/1220
- https://github.com/rancher/rancher-docs/pull/1221
- https://github.com/rancher/rancher-docs/pull/1211(Merged)
- https://github.com/rancher/rancher-docs/pull/1203(Merged)
- https://github.com/rancher/rancher-docs/pull/1199(Merged)

**Comment by martyav** [comment]
JSYK #1207 has a merge conflict. I think the issue is that the edits use 4 spaces whereas `main` use 2 spaces to indent on the same lines, but since I can't read the content I'll wait for a translator to resolve it.

**Pull request merged**